src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -521,6 +521,7 @@ public interface IStartConfiguration
             /// No update checks or analytics collection should be done.
             /// </summary>
             bool IsHeadless { get; set; }
+            IPathManager PathManager { get; set; }
```

Comment:
I dont think we can change these interface unfortunately, since these are singleton now, they can be accessed anywhere in Dynamo code, we dont need to pass them around

Reply:
My Intellisense was taking the Interface instead of the Class, Done.